### PR TITLE
Update bootstrap

### PR DESCRIPTION
**Pull request by serhii-lekariev**
This PR bumps the version of the Bootstrap plugin to the freshest one.
It also updates the version of `gradlew`.

All now-deprecated usage has been eliminated.

### CODE REVIEW

**Comment by serhii-lekariev** [comment]
@armiol, PTAL.

File: server/src/test/java/io/spine/examples/kanban/server/board/BoardInitProcessTest.java [hunk 1]
```diff
@@ -63,23 +65,30 @@ void issuesCreationCommands() {
...
-                 .deletedFlag()
-                 .isTrue();
+        context().assertEntity(board(), BoardInitProcess.class)
+                .deletedFlag()
+                .isTrue();
+    }
+
+    private Collection<ColumnId> createdColumns() {
+        List<ColumnId
+                > collect = context().assertCommands()
```
Comment (armiol) [inline server/src/test/java/io/spine/examples/kanban/server/board/BoardInitProcessTest.java:85]: Please fix.

File: server/src/test/java/io/spine/examples/kanban/server/board/BoardInitProcessTest.java [hunk 2]
```diff
@@ -63,23 +65,30 @@ void issuesCreationCommands() {
...
+                .withType(CreateColumn.class)
+                .actual()
+                .stream()
+                .map(c -> AnyPacker.unpack(c.getMessage(), CreateColumn.class))
```
Comment (armiol) [inline server/src/test/java/io/spine/examples/kanban/server/board/BoardInitProcessTest.java:89]: Let's use the static import here.

File: server/src/test/java/io/spine/examples/kanban/server/view/BoardProjectionTest.java [hunk 3]
```diff
@@ -112,7 +109,15 @@ void cards() {
...
+        return context().assertEntity(board(), BoardProjection.class)
                         .hasStateThat();
     }
+
+    private <T extends CommandMessage> Stream<T> commands(Class<T> commandClass) {
+        return context().assertCommands()
+                .actual()
+                .stream()
+                .filter(commandClass::isInstance)
+                .map(c -> AnyPacker.unpack(c.getMessage(), commandClass));
```
Comment (armiol) [inline server/src/test/java/io/spine/examples/kanban/server/view/BoardProjectionTest.java:121]: Same as above.

**Pull request merged**